docs/source/howto/ssh.rst

```diff
@@ -196,34 +196,67 @@ This section explains how to use the ``proxy_command`` feature of ``ssh`` in ord
 SSH configuration
 ^^^^^^^^^^^^^^^^^
 
-Edit the ``~/.ssh/config`` file on the computer on which you installed AiiDA (or create it if missing) and add the following lines::
+To decide whether to use the ``ProxyJump`` (recommended) or the ``ProxyCommand`` directive, please check the version of your SSH client first with ``ssh -V``.
+The ``ProxyJump`` directive has been added in version 7.3 of OpenSSH, hence if you are using an older version of SSH (on your machine or the *PROXY*) you have to use ``ProxyCommand``.
+
+To setup the proxy configuration with ``ProxyJump`, edit the ``~/.ssh/config`` file on the computer on which you installed AiiDA (or create it if missing)
+and add the following lines::
+
+  Host SHORTNAME_TARGET
+      Hostname FULLHOSTNAME_TARGET
+      User USER_TARGET
+      IdentityFile ~/.ssh/aiida
+      ProxyJump USER_PROXY@FULLHOSTNAME_PROXY
+
+  Host FULLHOSTNAME_PROXY
+      IdentityFile ~/.ssh/aiida
+
+replacing the ``..._TARGET`` and ``..._PROXY`` variables with the host/user names of the respective servers.
+
+To setup the proxy configuration with ``ProxyCommand` **instead**, edit the ``~/.ssh/config`` file on the computer on which you installed AiiDA (or create it if missing)
```

Comment:
In order not to overload the user with options, I would suggest to hide this inside a

```
.. dropdown:: :fa:`plus-circle` Alternative setup with ``ProxyCommand``
```

(not mandatory)